client/tsconfig.json

```diff
@@ -1,18 +1,21 @@
 {
     "compilerOptions": {
-        "target": "es6",
+        "target": "es5",
```

Comment:
we downgraded?

Reply:
We originally targeted es6 with the full stack (parser, formatter), but I remember there being a compatibility issue where we had to downgrade. I think this is the first time I've touched the language services layer in a while.